ui/src/components/policy/PolicyList.js

```diff
@@ -110,36 +200,47 @@ export default class PolicyList extends React.Component {
         });
     }
 
-    reloadPolicies(successMessage, showSuccess) {
-        let role = this.props.role;
+    onClickDuplicatePolicyVersion(
+        policyName,
+        version,
+        reloadPolicyVersionsFunc
+    ) {
+        this.setState({
+            showDuplicatePolicy: true,
+            duplicatePolicyName: policyName,
+            duplicateVersionSourceName: version,
+            reloadPolicyVersionsFunc: reloadPolicyVersionsFunc,
+            errorMessage: null,
+        });
+    }
 
+    closeModals(showSuccess, successMessage) {
```

Comment:
I think we can merge closeModals and closeModal to a single functions. Wherever we are calling closeModal(), we can send showSuccess as false to closeModals()

Reply:
fixed